network/p2p/acp118/handler_test.go

```diff
@@ -52,15 +52,7 @@ func TestHandler(t *testing.T) {
 			chainID := ids.GenerateTestID()
 			signer := warp.NewSigner(sk, networkID, chainID)
 			h := NewHandler(tt.verifier, signer)
-			clientNodeID := ids.GenerateTestNodeID()
-			serverNodeID := ids.GenerateTestNodeID()
-			c := p2ptest.NewClient(
-				t,
-				ctx,
-				h,
-				clientNodeID,
-				serverNodeID,
-			)
+			c := p2ptest.Client{ServerHandler: h}
```

Comment:
(style) This is too far from its usage to just be called `c`. I'd move the declaration to the end (as against renaming it) because I spent the whole time trying to understand where it's used. Some other variables (at least the handler) should also be addressed for the same reason.